### PR TITLE
[SYSDM] Add file hdrs. Improve pt-PT translation. Fix accel collisions. Add missing accell keys.

### PR DESCRIPTION
**Pull request by zecarlos1957**
Add file header.
Improve Portuguese (pt-PT) translation.
Fix accel collisions. Add missing accell keys.
Remove accel from Ok and Cancel buttons.

### CODE REVIEW

**Review by HBelusca — APPROVED** [review]

File: dll/cpl/sysdm/lang/pt-PT.rc [hunk 1]
```diff
@@ -242,23 +245,23 @@ BEGIN
     LTEXT "%s", IDC_DRIVE, 88, 80, 100, 9
     LTEXT "Espaço disponível:", IDC_STATIC, 20, 92, 60, 9
     LTEXT "%s", IDC_SPACEAVAIL, 88, 92, 104, 9
-    LTEXT "Tamanho inicial (MB):", -1, 22, 118, 75, 9
-    LTEXT "Tamanho máximo (MB):", -1, 22, 131, 75, 9
+    LTEXT "Tamanho &inicial (MB):", -1, 22, 118, 75, 9
```
Comment (JoachimHenze) [inline dll/cpl/sysdm/lang/pt-PT.rc:248]: MS really uses &i here? They do typically recommend to avoid that. Have a pic?

**Review by JoachimHenze — APPROVED** [review]
Rest aside the &i thingy LGTM. Approved.

File: dll/cpl/sysdm/lang/pt-PT.rc (hunk 1 above)
Comment (zecarlos1957) [reply dll/cpl/sysdm/lang/pt-PT.rc:248]: VirtualBox_WinSrv2003_Memoria

Yes, on Win2003 Server Enterprise. But in WinXP there is no such accelerator.
Do you want me to remove these 3 accelerators?

Comment (JoachimHenze) [reply dll/cpl/sysdm/lang/pt-PT.rc:248]: No don't remove the accels. But use "&Tamnho inicial" instead of Tamanho &initial" here. That avoids the ugly &i.

Comment (JoachimHenze) [inline dll/cpl/sysdm/lang/pt-PT.rc:248]:
```suggestion
    LTEXT "&Tamanho inicial (MB):", -1, 22, 118, 75, 9
```

**Pull request merged**